osmosis/assetlist.json

```diff
@@ -25755,6 +25755,41 @@
       "name": "Earn CDT Vault",
       "display": "earnCDT",
       "symbol": "earnCDT"
-    }
+    },
+    {
+      "description": "The Spice memecoin",
+      "denom_units": [
+        {
+          "denom": "factory/osmo1n6asrjy9754q8y9jsxqf557zmsv3s3xa5m9eg5/uspice",
+          "exponent": 0
+        },
+        {
+          "denom": "uspice",
```

Comment:
this denom is quite inappropriate, since the 'u-' prefix is often used for 'micro-', meaning, 1 millionth of something. This is the opposite. I recommend just 'spice' instead.